SU-HSDO/suhumsci
# SHS-5098: Update google fonts

# READY FOR REVIEW

## Summary
Update decanter to v6.2.19

## Need Review By (Date)
10/26

## Urgency
medium

## Steps to Test
- In the theme folder (`docroot/themes/humsci/humsci_basic`) run `rm -rf node_modules && npm install`
- In `traditional` and `colorful` sites, clear the Drupal cache and visit any page. Using the browser dev tools, confirm that Source Sans 3 and Source Serif 4 instead of Source Sans Pro and Source Serif Pro.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
